languages/js/src/Host.ts

```diff
@@ -89,6 +89,52 @@ export class Host {
     return cls;
   }
 
+  /**
+   * Get user type for `cls`.
+   */
+  getType(cls: Class):
```

Comment:
I don't really get why we needed to add this if we're not adding the full `UserTypes` thing yet, since it's not actually removing the host attributes (`classIds`, `classNames`, etc.).

Reply:
It seems like we only use this in a single place and then immediately dump everything except the returned `id` field. Should this be `getId` instead of `getType`?

Reply:
I added this so it would be easier to implement with the API Gwen is adding in #1116.